### DISCUSSION TITLE
Test Request Window Bugs  (Fastify)

### DISCUSSION THREAD

**Discussion by cristianabs** [scalar/scalar#1704] [Q&A]
Hello everyone, I have come across the following issues testing the Test Request window:

1. The Send Button disappear when long URIs

<img width="1643" alt="2" src="https://github.com/scalar/scalar/assets/6587063/bd0ea96d-ae65-4c88-8942-43a2521b07ee">
<img width="1638" alt="3" src="https://github.com/scalar/scalar/assets/6587063/ef5b7135-32f3-47b9-b86f-c108878648b4">
<img width="1641" alt="4" src="https://github.com/scalar/scalar/assets/6587063/c0625909-68fd-4a73-8b52-ee90af05a5d3">

2. The Response box does not show any response after sending the request, it appears between 5 to 10 minutes after, same behaviour as the "history" button.

<img width="1641" alt="1" src="https://github.com/scalar/scalar/assets/6587063/9150ec9a-605b-4675-8241-0185a0820638">
<img width="1710" alt="5" src="https://github.com/scalar/scalar/assets/6587063/f75480b0-fd1e-465c-bcd3-664b02f9cc05">
<img width="231" alt="6" src="https://github.com/scalar/scalar/assets/6587063/7c4dcda7-7165-4b59-968e-822b08dadafe">

I tested both using Firefox and Chrome and I am using the Fastify version.

Any ideas?

**Comment by marclave** [comment]
thanks for reporting this @cristianabs ! let's get this fixed :)

**Reply by cristianabs** [reply]
Hi @marclave I also noticed that when you send an API request, it doesn't work if you want to send a new one either the same URI or another from the list if you keep the same Test Request window.

**Reply by antlio** [reply]
thanks for reporting @cristianabs, the UI part should be fixed by [#1701](https://github.com/scalar/scalar/pull/1701), we'll update dependencies.

**Reply by marclave** [reply]
hi @cristianabs I wasnt able to recreate this bug you're saying, would you be able to do a screen recording or show example screenshots?

**Reply by cristianabs** [reply]
Hi @marclave here is the screen recording, let me know if it is enough or if I can do anything else.

I have uploaded the video on Gdrive as its size is more than 10MB.

https://drive.google.com/file/d/1TJkPGZl0Z4qaCERYjnCby5U-nyGjHEI6/view?usp=drive_link
